src/locale/utils.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import reactStringReplace from 'react-string-replace'
```

Comment:
have you considered using [that approach](https://stackoverflow.com/questions/39836369/html-tags-in-i18next-translation-files-in-react) instead?

Reply:
I didn't find that option! This is a great one.
The only problem is that in Hebrew it's not too comfortable/clear in the JSON file, dealing with text like: 
`יש פה טקסט עם <0/> לינק אחד <0> ועם <1/> עוד לינק<1>`
which is required in the JSON file.
(Another option to use the approach you suggested is to write the Hebrew version directly in the file. But I don't like it, since most of the text is already in the JSON file).

Will check for a better way of dealing with this and update the code accordingly.

Reply:
another valid approach is to separate the text into smaller parts

Reply:
Changed the files to use the tag you suggested